adhocracy-plus/assets/scss/_variables.scss

```diff
@@ -10,7 +10,7 @@ $brand-tertiary: #008dc5 !default; // prussian blue
 $brand-success: #a3ef90 !default;
 $brand-info: $brand-primary !default; // $brand-primary from platform.scss
 $brand-warning: #ffc107 !default;
-$brand-danger: #a72b1e !default;
+$brand-danger: #f00 !default;
 
```

Comment:
This variable is used for many elements, this effects all error messages and as we use lighten() for lots of them, changing it means that colour dissapears. If we need a new colour we can either define a new variable in this file or in the css file for whichever component we are working on, or I sometimes check with the designer if they think we should change it everywhere as it's best not to add lots of colours to a site. here I would leave it as is and check with design after it's on dev so they can see it.